src/states/bootstrapping_peer.rs

```diff
@@ -310,15 +268,9 @@ impl Base for BootstrappingPeer {
 
     fn handle_connected_to(
         &mut self,
-        conn_info: ConnectionInfo,
+        _conn_info: ConnectionInfo,
         _outbox: &mut dyn EventBox,
     ) -> Transition {
-        debug!(
-            "{} Received ConnectedTo event from {}.",
-            self, conn_info.peer_addr
-        );
-
-        self.send_bootstrap_request(conn_info);
```

Comment:
Can you explain why we did that, and we no longer need to?

Reply:
The previous flow was:
1. initiate connection to the bootstrapping nodes
2. wait for `ConnectedTo` from them
3. send `BootstrapRequest` when we receive it

Now this changes to simply:
1. send `BootstrapRequest` 

So we don't need to do anything in `handle_connected_to` anymore.